Makefile

```diff
@@ -92,8 +95,13 @@ toolname:
 	@$(SCRIPTS_DIR)/show_toolchains.sh $(FULL_PATH) $(BR_VER)
 
 list-configs:
+ifndef BOARD
+	@echo "Variable BOARD must be defined to list configs"
```

Comment:
then maybe `$(error Variable BOARD must be defined to initialize output directory)` instead, so that it reports the error and exits.